2022-06-30 00:31:23,898 DEBUG scheduler Access denied for site test
2022-06-30 00:32:24,404 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:32:24,521 DEBUG scheduler Access denied for site test
2022-06-30 00:32:24,667 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:32:24,747 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:32:24,987 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:32:25,141 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:32:25,371 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:32:25,456 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:32:25,728 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:32:25,811 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext
2022-06-30 00:32:26,007 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext
2022-06-30 00:32:26,092 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:32:26,267 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:32:26,348 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:32:26,554 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:32:26,631 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:32:26,869 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:32:26,953 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:32:27,240 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:32:27,319 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext
2022-06-30 00:32:27,514 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext
2022-06-30 00:33:25,279 DEBUG scheduler Access denied for site test
2022-06-30 00:34:26,071 DEBUG scheduler Access denied for site test
2022-06-30 00:35:27,020 DEBUG scheduler Access denied for site test
2022-06-30 00:36:28,228 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:36:28,460 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:36:28,537 DEBUG scheduler Access denied for site test
2022-06-30 00:36:28,558 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:36:28,793 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:36:28,894 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:36:29,086 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:36:29,194 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:36:29,516 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:36:29,608 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext
2022-06-30 00:36:29,786 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext
2022-06-30 00:36:29,914 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:36:30,107 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:36:30,200 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:36:30,391 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:36:30,478 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:36:30,720 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:36:30,819 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:36:31,058 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:36:31,167 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext
2022-06-30 00:36:31,370 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext
2022-06-30 00:37:29,237 DEBUG scheduler Access denied for site test
2022-06-30 00:38:29,704 DEBUG scheduler Access denied for site test
2022-06-30 00:39:30,296 DEBUG scheduler Access denied for site test
2022-06-30 00:40:31,084 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:40:31,347 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:40:31,385 DEBUG scheduler Access denied for site test
2022-06-30 00:40:31,441 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:40:31,666 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:40:31,759 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:40:31,939 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:40:32,028 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:40:32,314 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:40:32,391 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext
2022-06-30 00:40:32,565 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext
2022-06-30 00:40:32,638 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:40:32,804 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:40:32,892 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:40:33,096 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:40:33,177 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:40:33,420 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:40:33,507 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:40:33,789 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:40:33,879 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext
2022-06-30 00:40:34,131 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext
2022-06-30 00:41:32,017 DEBUG scheduler Access denied for site test
2022-06-30 00:42:33,056 DEBUG scheduler Access denied for site test
2022-06-30 00:43:34,157 DEBUG scheduler Access denied for site test
2022-06-30 00:44:35,109 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:44:35,368 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:44:35,474 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:44:35,683 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:44:35,690 DEBUG scheduler Access denied for site test
2022-06-30 00:44:35,787 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:44:35,975 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:44:36,060 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:44:36,357 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:44:36,449 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext
2022-06-30 00:44:36,646 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext
2022-06-30 00:44:36,730 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:44:36,892 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:44:36,985 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:44:37,188 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:44:37,290 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:44:37,529 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:44:37,629 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:44:37,921 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:44:38,024 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext
2022-06-30 00:44:38,266 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext
2022-06-30 00:45:36,140 DEBUG scheduler Access denied for site test
2022-06-30 00:45:36,200 INFO scheduler Scheduled Job Start: frappe.twofactor.delete_all_barcodes_for_users for erpnext
2022-06-30 00:45:36,446 INFO scheduler Scheduled Job Complete: frappe.twofactor.delete_all_barcodes_for_users for erpnext
2022-06-30 00:45:36,641 INFO scheduler Scheduled Job Start: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext
2022-06-30 00:45:36,921 INFO scheduler Scheduled Job Complete: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext
2022-06-30 00:45:37,014 INFO scheduler Scheduled Job Start: frappe.oauth.delete_oauth2_data for erpnext
2022-06-30 00:45:37,273 INFO scheduler Scheduled Job Complete: frappe.oauth.delete_oauth2_data for erpnext
2022-06-30 00:46:36,767 DEBUG scheduler Access denied for site test
2022-06-30 00:47:38,010 DEBUG scheduler Access denied for site test
2022-06-30 00:48:38,746 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:48:39,005 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 00:48:39,086 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:48:39,109 DEBUG scheduler Access denied for site test
2022-06-30 00:48:39,311 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 00:48:39,399 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:48:39,603 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 00:48:39,699 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:48:40,018 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 00:48:40,119 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext
2022-06-30 00:48:40,332 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext
2022-06-30 00:48:40,433 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:48:40,610 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 00:48:40,692 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:48:40,915 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 00:48:40,995 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:48:41,237 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 00:48:41,338 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:48:41,615 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 00:48:41,718 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext
2022-06-30 00:48:41,940 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext
                                                                    2022-06-30 08:16:33,618 ERROR scheduler Exception in Enqueue Events for Site demo
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:16:33,624 ERROR scheduler Exception in Enqueue Events for Site erpnext
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:16:33,628 ERROR scheduler Exception in Enqueue Events for Site test
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:16:33,632 ERROR scheduler Exception in Enqueue Events for Site test.dev
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:16:33,637 ERROR scheduler Exception in Enqueue Events for Site test1
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:17:34,463 ERROR scheduler Exception in Enqueue Events for Site demo
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:17:34,467 ERROR scheduler Exception in Enqueue Events for Site erpnext
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:17:34,468 ERROR scheduler Exception in Enqueue Events for Site test
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:17:34,470 ERROR scheduler Exception in Enqueue Events for Site test.dev
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:17:34,472 ERROR scheduler Exception in Enqueue Events for Site test1
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:18:34,639 ERROR scheduler Exception in Enqueue Events for Site demo
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:18:34,640 ERROR scheduler Exception in Enqueue Events for Site erpnext
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:18:34,642 ERROR scheduler Exception in Enqueue Events for Site test
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:18:34,643 ERROR scheduler Exception in Enqueue Events for Site test.dev
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:18:34,644 ERROR scheduler Exception in Enqueue Events for Site test1
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:19:35,590 ERROR scheduler Exception in Enqueue Events for Site demo
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:19:35,592 ERROR scheduler Exception in Enqueue Events for Site erpnext
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:19:35,593 ERROR scheduler Exception in Enqueue Events for Site test
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:19:35,595 ERROR scheduler Exception in Enqueue Events for Site test.dev
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:19:35,596 ERROR scheduler Exception in Enqueue Events for Site test1
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:20:36,038 ERROR scheduler Exception in Enqueue Events for Site demo
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:20:36,040 ERROR scheduler Exception in Enqueue Events for Site erpnext
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:20:36,056 ERROR scheduler Exception in Enqueue Events for Site test
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:20:36,057 ERROR scheduler Exception in Enqueue Events for Site test.dev
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:20:36,058 ERROR scheduler Exception in Enqueue Events for Site test1
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:21:36,434 ERROR scheduler Exception in Enqueue Events for Site demo
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:21:36,436 ERROR scheduler Exception in Enqueue Events for Site erpnext
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:21:36,437 ERROR scheduler Exception in Enqueue Events for Site test
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:21:36,438 ERROR scheduler Exception in Enqueue Events for Site test.dev
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:21:36,440 ERROR scheduler Exception in Enqueue Events for Site test1
Traceback (most recent call last):
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 613, in connect
    sock = socket.create_connection(
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/opt/bitnami/python/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 70, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 105, in is_scheduler_inactive
    if is_scheduler_disabled():
  File "apps/frappe/frappe/utils/scheduler.py", line 115, in is_scheduler_disabled
    return not frappe.utils.cint(frappe.db.get_single_value("System Settings", "enable_scheduler"))
  File "apps/frappe/frappe/database/database.py", line 671, in get_single_value
    val = self.sql(
  File "apps/frappe/frappe/database/database.py", line 148, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 84, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 76, in get_connection
    conn = pymysql.connect(
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 353, in __init__
    self.connect()
  File "env/lib/python3.8/site-packages/pymysql/connections.py", line 664, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2022-06-30 08:23:57,720 DEBUG scheduler Access denied for site demo
2022-06-30 08:23:58,640 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_daily_feedback_reminder for erpnext
2022-06-30 08:23:58,640 INFO scheduler Scheduled Job Start: erpnext.stock.doctype.repost_item_valuation.repost_item_valuation.repost_entries for erpnext
2022-06-30 08:24:00,545 INFO scheduler Scheduled Job Complete: erpnext.stock.doctype.repost_item_valuation.repost_item_valuation.repost_entries for erpnext
2022-06-30 08:24:00,558 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_daily_feedback_reminder for erpnext
2022-06-30 08:24:00,671 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.lead.lead.daily_open_lead for erpnext
2022-06-30 08:24:00,781 INFO scheduler Scheduled Job Start: erpnext.non_profit.doctype.membership.membership.set_expired_status for erpnext
2022-06-30 08:24:00,888 DEBUG scheduler Queued events for site erpnext
2022-06-30 08:24:00,894 DEBUG scheduler Access denied for site test
2022-06-30 08:24:00,899 DEBUG scheduler Access denied for site test.dev
2022-06-30 08:24:01,134 INFO scheduler Scheduled Job Complete: erpnext.non_profit.doctype.membership.membership.set_expired_status for erpnext
2022-06-30 08:24:01,325 INFO scheduler Scheduled Job Start: erpnext.accounts.doctype.process_statement_of_accounts.process_statement_of_accounts.send_auto_email for erpnext
2022-06-30 08:24:01,604 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.lead.lead.daily_open_lead for erpnext
2022-06-30 08:24:01,748 INFO scheduler Scheduled Job Start: erpnext.loan_management.doctype.process_loan_interest_accrual.process_loan_interest_accrual.process_loan_interest_accrual_for_term_loans for erpnext
2022-06-30 08:24:01,995 INFO scheduler Scheduled Job Complete: erpnext.accounts.doctype.process_statement_of_accounts.process_statement_of_accounts.send_auto_email for erpnext
2022-06-30 08:24:02,107 INFO scheduler Scheduled Job Start: erpnext.buying.doctype.supplier_quotation.supplier_quotation.set_expired_status for erpnext
2022-06-30 08:24:02,174 INFO scheduler Scheduled Job Complete: erpnext.loan_management.doctype.process_loan_interest_accrual.process_loan_interest_accrual.process_loan_interest_accrual_for_term_loans for erpnext
2022-06-30 08:24:02,431 INFO scheduler Scheduled Job Start: erpnext.loan_management.doctype.process_loan_security_shortfall.process_loan_security_shortfall.create_process_loan_security_shortfall for erpnext
2022-06-30 08:24:02,507 INFO scheduler Scheduled Job Complete: erpnext.buying.doctype.supplier_quotation.supplier_quotation.set_expired_status for erpnext
2022-06-30 08:24:02,609 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.update_appointment_status for erpnext
2022-06-30 08:24:02,725 INFO scheduler Scheduled Job Complete: erpnext.loan_management.doctype.process_loan_security_shortfall.process_loan_security_shortfall.create_process_loan_security_shortfall for erpnext
2022-06-30 08:24:02,835 INFO scheduler Scheduled Job Start: erpnext.hr.utils.allocate_earned_leaves for erpnext
2022-06-30 08:24:02,920 INFO scheduler Scheduled Job Failed: erpnext.healthcare.doctype.patient_appointment.patient_appointment.update_appointment_status for erpnext
2022-06-30 08:24:03,037 INFO scheduler Scheduled Job Start: erpnext.selling.doctype.quotation.quotation.set_expired_status for erpnext
2022-06-30 08:24:03,208 INFO scheduler Scheduled Job Complete: erpnext.hr.utils.allocate_earned_leaves for erpnext
2022-06-30 08:24:03,386 INFO scheduler Scheduled Job Start: erpnext.hr.utils.generate_leave_encashment for erpnext
2022-06-30 08:24:03,656 INFO scheduler Scheduled Job Complete: erpnext.selling.doctype.quotation.quotation.set_expired_status for erpnext
2022-06-30 08:24:03,756 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.email_campaign.email_campaign.set_email_campaign_status for erpnext
2022-06-30 08:24:03,807 INFO scheduler Scheduled Job Complete: erpnext.hr.utils.generate_leave_encashment for erpnext
2022-06-30 08:24:03,925 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.leave_ledger_entry.leave_ledger_entry.process_expired_allocation for erpnext
2022-06-30 08:24:04,990 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.leave_ledger_entry.leave_ledger_entry.process_expired_allocation for erpnext
2022-06-30 08:24:04,993 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.email_campaign.email_campaign.set_email_campaign_status for erpnext
2022-06-30 08:24:05,101 INFO scheduler Scheduled Job Start: erpnext.manufacturing.doctype.bom_update_tool.bom_update_tool.auto_update_latest_price_in_all_boms for erpnext
2022-06-30 08:24:05,168 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.email_campaign.email_campaign.send_email_to_leads_or_contacts for erpnext
2022-06-30 08:24:05,532 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.email_campaign.email_campaign.send_email_to_leads_or_contacts for erpnext
2022-06-30 08:24:05,661 INFO scheduler Scheduled Job Start: erpnext.support.doctype.service_level_agreement.service_level_agreement.check_agreement_status for erpnext
2022-06-30 08:24:05,704 INFO scheduler Scheduled Job Complete: erpnext.manufacturing.doctype.bom_update_tool.bom_update_tool.auto_update_latest_price_in_all_boms for erpnext
2022-06-30 08:24:05,833 INFO scheduler Scheduled Job Start: erpnext.setup.doctype.email_digest.email_digest.send for erpnext
2022-06-30 08:24:06,103 INFO scheduler Scheduled Job Complete: erpnext.support.doctype.service_level_agreement.service_level_agreement.check_agreement_status for erpnext
2022-06-30 08:24:06,217 INFO scheduler Scheduled Job Start: erpnext.quality_management.doctype.quality_review.quality_review.review for erpnext
2022-06-30 08:24:06,718 INFO scheduler Scheduled Job Failed: erpnext.setup.doctype.email_digest.email_digest.send for erpnext
2022-06-30 08:24:06,725 INFO scheduler Scheduled Job Complete: erpnext.quality_management.doctype.quality_review.quality_review.review for erpnext
2022-06-30 08:24:06,892 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.google_drive.google_drive.weekly_backup for erpnext
2022-06-30 08:24:06,920 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.send_project_status_email_to_users for erpnext
2022-06-30 08:24:07,493 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.send_project_status_email_to_users for erpnext
2022-06-30 08:24:07,605 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.update_project_sales_billing for erpnext
2022-06-30 08:24:07,792 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.google_drive.google_drive.weekly_backup for erpnext
2022-06-30 08:24:07,910 INFO scheduler Scheduled Job Start: frappe.social.doctype.energy_point_log.energy_point_log.send_weekly_summary for erpnext
2022-06-30 08:24:08,079 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.update_project_sales_billing for erpnext
2022-06-30 08:24:08,186 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.contract.contract.update_status_for_contracts for erpnext
2022-06-30 08:24:08,207 INFO scheduler Scheduled Job Complete: frappe.social.doctype.energy_point_log.energy_point_log.send_weekly_summary for erpnext
2022-06-30 08:24:08,296 INFO scheduler Scheduled Job Start: frappe.desk.form.document_follow.send_weekly_updates for erpnext
2022-06-30 08:24:08,574 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.contract.contract.update_status_for_contracts for erpnext
2022-06-30 08:24:08,588 INFO scheduler Scheduled Job Complete: frappe.desk.form.document_follow.send_weekly_updates for erpnext
2022-06-30 08:24:08,665 INFO scheduler Scheduled Job Start: erpnext.assets.doctype.asset.asset.make_post_gl_entry for erpnext
2022-06-30 08:24:08,681 INFO scheduler Scheduled Job Start: frappe.desk.doctype.route_history.route_history.flush_old_route_records for erpnext
2022-06-30 08:24:09,101 INFO scheduler Scheduled Job Complete: frappe.desk.doctype.route_history.route_history.flush_old_route_records for erpnext
2022-06-30 08:24:09,133 INFO scheduler Scheduled Job Complete: erpnext.assets.doctype.asset.asset.make_post_gl_entry for erpnext
2022-06-30 08:24:09,207 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.s3_backup_settings.s3_backup_settings.take_backups_weekly for erpnext
2022-06-30 08:24:09,269 INFO scheduler Scheduled Job Start: erpnext.assets.doctype.asset.asset.update_maintenance_status for erpnext
2022-06-30 08:24:09,651 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.s3_backup_settings.s3_backup_settings.take_backups_weekly for erpnext
2022-06-30 08:24:09,753 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.dropbox_settings.dropbox_settings.take_backups_weekly for erpnext
2022-06-30 08:24:09,951 INFO scheduler Scheduled Job Complete: erpnext.assets.doctype.asset.asset.update_maintenance_status for erpnext
2022-06-30 08:24:10,149 INFO scheduler Scheduled Job Start: erpnext.setup.doctype.company.company.cache_companies_monthly_sales_history for erpnext
2022-06-30 08:24:10,998 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.dropbox_settings.dropbox_settings.take_backups_weekly for erpnext
2022-06-30 08:24:11,084 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.google_drive.google_drive.daily_backup for erpnext
2022-06-30 08:24:11,141 INFO scheduler Scheduled Job Complete: erpnext.setup.doctype.company.company.cache_companies_monthly_sales_history for erpnext
2022-06-30 08:24:11,353 INFO scheduler Scheduled Job Start: erpnext.buying.doctype.supplier_scorecard.supplier_scorecard.refresh_scorecards for erpnext
2022-06-30 08:24:11,369 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.google_drive.google_drive.daily_backup for erpnext
2022-06-30 08:24:11,467 INFO scheduler Scheduled Job Start: frappe.email.doctype.auto_email_report.auto_email_report.send_daily for erpnext
2022-06-30 08:24:11,591 INFO scheduler Scheduled Job Complete: erpnext.buying.doctype.supplier_scorecard.supplier_scorecard.refresh_scorecards for erpnext
2022-06-30 08:24:11,675 INFO scheduler Scheduled Job Start: erpnext.stock.doctype.serial_no.serial_no.update_maintenance_status for erpnext
2022-06-30 08:24:12,108 INFO scheduler Scheduled Job Complete: erpnext.stock.doctype.serial_no.serial_no.update_maintenance_status for erpnext
2022-06-30 08:24:12,184 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.daily_work_summary_group.daily_work_summary_group.send_summary for erpnext
2022-06-30 08:24:12,436 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.daily_work_summary_group.daily_work_summary_group.send_summary for erpnext
2022-06-30 08:24:12,516 INFO scheduler Scheduled Job Start: erpnext.assets.doctype.asset.depreciation.post_depreciation_entries for erpnext
2022-06-30 08:24:12,568 INFO scheduler Scheduled Job Complete: frappe.email.doctype.auto_email_report.auto_email_report.send_daily for erpnext
2022-06-30 08:24:12,659 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.s3_backup_settings.s3_backup_settings.take_backups_daily for erpnext
2022-06-30 08:24:12,728 INFO scheduler Scheduled Job Complete: erpnext.assets.doctype.asset.depreciation.post_depreciation_entries for erpnext
2022-06-30 08:24:12,925 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.task.task.set_tasks_as_overdue for erpnext
2022-06-30 08:24:12,986 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.s3_backup_settings.s3_backup_settings.take_backups_daily for erpnext
2022-06-30 08:24:13,173 INFO scheduler Scheduled Job Start: frappe.utils.change_log.check_for_update for erpnext
2022-06-30 08:24:13,247 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.task.task.set_tasks_as_overdue for erpnext
2022-06-30 08:24:13,362 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.employee.employee_reminders.send_birthday_reminders for erpnext
2022-06-30 08:24:13,492 INFO scheduler Scheduled Job Complete: frappe.utils.change_log.check_for_update for erpnext
2022-06-30 08:24:13,636 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.employee.employee_reminders.send_birthday_reminders for erpnext
2022-06-30 08:24:13,651 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.dropbox_settings.dropbox_settings.take_backups_daily for erpnext
2022-06-30 08:24:13,733 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.employee.employee_reminders.send_work_anniversary_reminders for erpnext
2022-06-30 08:24:14,019 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.employee.employee_reminders.send_work_anniversary_reminders for erpnext
2022-06-30 08:24:14,076 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.dropbox_settings.dropbox_settings.take_backups_daily for erpnext
2022-06-30 08:24:14,111 INFO scheduler Scheduled Job Start: erpnext.accounts.doctype.fiscal_year.fiscal_year.auto_create_fiscal_year for erpnext
2022-06-30 08:24:14,318 INFO scheduler Scheduled Job Complete: erpnext.accounts.doctype.fiscal_year.fiscal_year.auto_create_fiscal_year for erpnext
2022-06-30 08:24:14,403 INFO scheduler Scheduled Job Start: erpnext.controllers.accounts_controller.update_invoice_status for erpnext
2022-06-30 08:24:14,691 INFO scheduler Scheduled Job Complete: erpnext.controllers.accounts_controller.update_invoice_status for erpnext
2022-06-30 08:24:14,776 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.opportunity.opportunity.auto_close_opportunity for erpnext
2022-06-30 08:24:15,030 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.opportunity.opportunity.auto_close_opportunity for erpnext
2022-06-30 08:24:15,124 INFO scheduler Scheduled Job Start: erpnext.support.doctype.issue.issue.auto_close_tickets for erpnext
2022-06-30 08:24:15,362 INFO scheduler Scheduled Job Complete: erpnext.support.doctype.issue.issue.auto_close_tickets for erpnext
2022-06-30 08:24:15,460 INFO scheduler Scheduled Job Start: erpnext.stock.reorder_item.reorder_item for erpnext
2022-06-30 08:24:15,791 INFO scheduler Scheduled Job Complete: erpnext.stock.reorder_item.reorder_item for erpnext
2022-06-30 08:24:15,892 INFO scheduler Scheduled Job Start: erpnext.erpnext_integrations.connectors.shopify_connection.sync_old_orders for erpnext
2022-06-30 08:24:16,462 INFO scheduler Scheduled Job Complete: erpnext.erpnext_integrations.connectors.shopify_connection.sync_old_orders for erpnext
2022-06-30 08:24:16,596 INFO scheduler Scheduled Job Start: erpnext.support.doctype.issue.issue.set_service_level_agreement_variance for erpnext
2022-06-30 08:24:17,001 INFO scheduler Scheduled Job Complete: erpnext.support.doctype.issue.issue.set_service_level_agreement_variance for erpnext
2022-06-30 08:24:17,139 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.shift_type.shift_type.process_auto_attendance_for_all_shifts for erpnext
2022-06-30 08:24:17,455 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.shift_type.shift_type.process_auto_attendance_for_all_shifts for erpnext
2022-06-30 08:24:17,552 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.collect_project_status for erpnext
2022-06-30 08:24:17,846 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.collect_project_status for erpnext
2022-06-30 08:24:17,940 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.hourly_reminder for erpnext
2022-06-30 08:24:18,278 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.hourly_reminder for erpnext
2022-06-30 08:24:18,370 INFO scheduler Scheduled Job Start: erpnext.erpnext_integrations.doctype.plaid_settings.plaid_settings.automatic_synchronization for erpnext
2022-06-30 08:24:18,798 INFO scheduler Scheduled Job Complete: erpnext.erpnext_integrations.doctype.plaid_settings.plaid_settings.automatic_synchronization for erpnext
2022-06-30 08:24:18,891 INFO scheduler Scheduled Job Start: erpnext.accounts.doctype.gl_entry.gl_entry.rename_gle_sle_docs for erpnext
2022-06-30 08:24:19,240 INFO scheduler Scheduled Job Complete: erpnext.accounts.doctype.gl_entry.gl_entry.rename_gle_sle_docs for erpnext
2022-06-30 08:24:19,340 INFO scheduler Scheduled Job Start: erpnext.erpnext_integrations.doctype.amazon_mws_settings.amazon_mws_settings.schedule_get_order_details for erpnext
2022-06-30 08:24:19,655 INFO scheduler Scheduled Job Complete: erpnext.erpnext_integrations.doctype.amazon_mws_settings.amazon_mws_settings.schedule_get_order_details for erpnext
2022-06-30 08:24:19,751 INFO scheduler Scheduled Job Start: erpnext.accounts.doctype.subscription.subscription.process_all for erpnext
2022-06-30 08:24:21,483 INFO scheduler Scheduled Job Complete: erpnext.accounts.doctype.subscription.subscription.process_all for erpnext
2022-06-30 08:24:21,582 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.daily_work_summary_group.daily_work_summary_group.trigger_emails for erpnext
2022-06-30 08:24:21,861 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.daily_work_summary_group.daily_work_summary_group.trigger_emails for erpnext
2022-06-30 08:24:21,960 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 08:24:22,214 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 08:24:22,300 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 08:24:22,508 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 08:24:22,601 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 08:24:22,850 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 08:24:22,949 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 08:24:23,226 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 08:24:23,316 INFO scheduler Scheduled Job Start: erpnext.utilities.doctype.video.video.update_youtube_data for erpnext
2022-06-30 08:24:23,757 INFO scheduler Scheduled Job Complete: erpnext.utilities.doctype.video.video.update_youtube_data for erpnext
2022-06-30 08:24:23,849 INFO scheduler Scheduled Job Start: frappe.core.doctype.log_settings.log_settings.run_log_clean_up for erpnext
2022-06-30 08:24:24,139 INFO scheduler Scheduled Job Complete: frappe.core.doctype.log_settings.log_settings.run_log_clean_up for erpnext
2022-06-30 08:24:24,228 INFO scheduler Scheduled Job Start: frappe.core.doctype.prepared_report.prepared_report.delete_expired_prepared_reports for erpnext
2022-06-30 08:24:24,519 INFO scheduler Scheduled Job Complete: frappe.core.doctype.prepared_report.prepared_report.delete_expired_prepared_reports for erpnext
2022-06-30 08:24:24,621 INFO scheduler Scheduled Job Start: frappe.email.doctype.unhandled_email.unhandled_email.remove_old_unhandled_emails for erpnext
2022-06-30 08:24:24,855 INFO scheduler Scheduled Job Complete: frappe.email.doctype.unhandled_email.unhandled_email.remove_old_unhandled_emails for erpnext
2022-06-30 08:24:24,959 INFO scheduler Scheduled Job Start: frappe.automation.doctype.auto_repeat.auto_repeat.set_auto_repeat_as_completed for erpnext
2022-06-30 08:24:25,314 INFO scheduler Scheduled Job Complete: frappe.automation.doctype.auto_repeat.auto_repeat.set_auto_repeat_as_completed for erpnext
2022-06-30 08:24:25,403 INFO scheduler Scheduled Job Start: frappe.automation.doctype.auto_repeat.auto_repeat.make_auto_repeat_entry for erpnext
2022-06-30 08:24:25,697 INFO scheduler Scheduled Job Complete: frappe.automation.doctype.auto_repeat.auto_repeat.make_auto_repeat_entry for erpnext
2022-06-30 08:24:25,786 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.google_contacts.google_contacts.sync for erpnext
2022-06-30 08:24:26,205 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.google_contacts.google_contacts.sync for erpnext
2022-06-30 08:24:26,297 INFO scheduler Scheduled Job Start: frappe.social.doctype.energy_point_settings.energy_point_settings.allocate_review_points for erpnext
2022-06-30 08:24:26,616 INFO scheduler Scheduled Job Complete: frappe.social.doctype.energy_point_settings.energy_point_settings.allocate_review_points for erpnext
2022-06-30 08:24:26,703 INFO scheduler Scheduled Job Start: frappe.desk.form.document_follow.send_daily_updates for erpnext
2022-06-30 08:24:26,951 INFO scheduler Scheduled Job Complete: frappe.desk.form.document_follow.send_daily_updates for erpnext
2022-06-30 08:24:27,043 INFO scheduler Scheduled Job Start: frappe.website.doctype.personal_data_deletion_request.personal_data_deletion_request.remove_unverified_record for erpnext
2022-06-30 08:24:27,271 INFO scheduler Scheduled Job Complete: frappe.website.doctype.personal_data_deletion_request.personal_data_deletion_request.remove_unverified_record for erpnext
2022-06-30 08:24:27,359 INFO scheduler Scheduled Job Start: frappe.utils.scheduler.restrict_scheduler_events_if_dormant for erpnext
2022-06-30 08:24:27,568 INFO scheduler Scheduled Job Failed: frappe.utils.scheduler.restrict_scheduler_events_if_dormant for erpnext
2022-06-30 08:24:27,655 INFO scheduler Scheduled Job Start: frappe.email.doctype.notification.notification.trigger_daily_alerts for erpnext
2022-06-30 08:24:27,952 INFO scheduler Scheduled Job Complete: frappe.email.doctype.notification.notification.trigger_daily_alerts for erpnext
2022-06-30 08:24:28,039 INFO scheduler Scheduled Job Start: frappe.sessions.clear_expired_sessions for erpnext
2022-06-30 08:24:28,242 INFO scheduler Scheduled Job Complete: frappe.sessions.clear_expired_sessions for erpnext
2022-06-30 08:24:28,323 INFO scheduler Scheduled Job Start: frappe.desk.doctype.event.event.send_event_digest for erpnext
2022-06-30 08:24:28,575 INFO scheduler Scheduled Job Complete: frappe.desk.doctype.event.event.send_event_digest for erpnext
2022-06-30 08:24:28,668 INFO scheduler Scheduled Job Start: frappe.core.doctype.error_log.error_log.set_old_logs_as_seen for erpnext
2022-06-30 08:24:28,887 INFO scheduler Scheduled Job Complete: frappe.core.doctype.error_log.error_log.set_old_logs_as_seen for erpnext
2022-06-30 08:24:28,977 INFO scheduler Scheduled Job Start: frappe.desk.notifications.clear_notifications for erpnext
2022-06-30 08:24:29,293 INFO scheduler Scheduled Job Complete: frappe.desk.notifications.clear_notifications for erpnext
2022-06-30 08:24:29,397 INFO scheduler Scheduled Job Start: frappe.email.queue.set_expiry_for_email_queue for erpnext
2022-06-30 08:24:29,660 INFO scheduler Scheduled Job Complete: frappe.email.queue.set_expiry_for_email_queue for erpnext
2022-06-30 08:24:29,755 INFO scheduler Scheduled Job Start: frappe.email.doctype.newsletter.newsletter.send_scheduled_email for erpnext
2022-06-30 08:24:30,140 INFO scheduler Scheduled Job Complete: frappe.email.doctype.newsletter.newsletter.send_scheduled_email for erpnext
2022-06-30 08:24:30,247 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.google_calendar.google_calendar.sync for erpnext
2022-06-30 08:24:30,734 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.google_calendar.google_calendar.sync for erpnext
2022-06-30 08:24:30,826 INFO scheduler Scheduled Job Start: frappe.desk.form.document_follow.send_hourly_updates for erpnext
2022-06-30 08:24:31,078 INFO scheduler Scheduled Job Complete: frappe.desk.form.document_follow.send_hourly_updates for erpnext
2022-06-30 08:24:31,236 INFO scheduler Scheduled Job Start: frappe.deferred_insert.save_to_db for erpnext
2022-06-30 08:24:31,529 INFO scheduler Scheduled Job Complete: frappe.deferred_insert.save_to_db for erpnext
2022-06-30 08:24:31,651 INFO scheduler Scheduled Job Start: frappe.desk.page.backups.backups.delete_downloadable_backups for erpnext
2022-06-30 08:24:32,012 INFO scheduler Scheduled Job Complete: frappe.desk.page.backups.backups.delete_downloadable_backups for erpnext
2022-06-30 08:24:32,134 INFO scheduler Scheduled Job Start: frappe.utils.error.collect_error_snapshots for erpnext
2022-06-30 08:24:52,233 INFO scheduler Scheduled Job Complete: frappe.utils.error.collect_error_snapshots for erpnext
2022-06-30 08:24:52,314 INFO scheduler Scheduled Job Start: frappe.model.utils.user_settings.sync_user_settings for erpnext
2022-06-30 08:24:52,504 INFO scheduler Scheduled Job Complete: frappe.model.utils.user_settings.sync_user_settings for erpnext
2022-06-30 08:24:52,579 INFO scheduler Scheduled Job Start: frappe.model.utils.link_count.update_link_count for erpnext
2022-06-30 08:24:52,767 INFO scheduler Scheduled Job Complete: frappe.model.utils.link_count.update_link_count for erpnext
2022-06-30 08:24:52,844 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext
2022-06-30 08:24:52,990 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext
2022-06-30 08:24:53,062 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 08:24:53,218 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 08:24:53,302 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 08:24:53,644 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 08:24:53,733 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 08:24:54,017 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 08:24:54,093 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 08:24:54,302 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 08:24:54,378 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext
2022-06-30 08:24:54,559 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext
2022-06-30 08:24:54,647 INFO scheduler Scheduled Job Start: frappe.twofactor.delete_all_barcodes_for_users for erpnext
2022-06-30 08:24:54,897 INFO scheduler Scheduled Job Complete: frappe.twofactor.delete_all_barcodes_for_users for erpnext
2022-06-30 08:24:54,990 INFO scheduler Scheduled Job Start: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext
2022-06-30 08:24:55,289 INFO scheduler Scheduled Job Complete: frappe.website.doctype.web_page.web_page.check_publish_status for erpnext
2022-06-30 08:24:55,371 INFO scheduler Scheduled Job Start: frappe.oauth.delete_oauth2_data for erpnext
2022-06-30 08:24:55,774 INFO scheduler Scheduled Job Complete: frappe.oauth.delete_oauth2_data for erpnext
2022-06-30 08:25:02,298 DEBUG scheduler Access denied for site demo
2022-06-30 08:25:02,497 DEBUG scheduler Queued events for site erpnext
2022-06-30 08:25:02,499 DEBUG scheduler Access denied for site test
2022-06-30 08:25:02,501 DEBUG scheduler Access denied for site test.dev
2022-06-30 08:26:03,498 DEBUG scheduler Access denied for site demo
2022-06-30 08:26:03,723 DEBUG scheduler Queued events for site erpnext
2022-06-30 08:26:03,726 DEBUG scheduler Access denied for site test
2022-06-30 08:26:03,729 DEBUG scheduler Access denied for site test.dev
2022-06-30 08:27:04,651 DEBUG scheduler Access denied for site demo
2022-06-30 08:27:04,851 DEBUG scheduler Queued events for site erpnext
2022-06-30 08:27:04,853 DEBUG scheduler Access denied for site test
2022-06-30 08:27:04,855 DEBUG scheduler Access denied for site test.dev
2022-06-30 08:28:05,542 DEBUG scheduler Access denied for site demo
2022-06-30 08:28:05,757 INFO scheduler Scheduled Job Start: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 08:28:05,775 DEBUG scheduler Queued events for site erpnext
2022-06-30 08:28:05,778 DEBUG scheduler Access denied for site test
2022-06-30 08:28:05,780 DEBUG scheduler Access denied for site test.dev
2022-06-30 08:28:06,008 INFO scheduler Scheduled Job Complete: erpnext.crm.doctype.social_media_post.social_media_post.process_scheduled_social_media_posts for erpnext
2022-06-30 08:28:06,083 INFO scheduler Scheduled Job Start: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 08:28:06,263 INFO scheduler Scheduled Job Complete: erpnext.hr.doctype.interview.interview.send_interview_reminder for erpnext
2022-06-30 08:28:06,336 INFO scheduler Scheduled Job Start: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 08:28:06,525 INFO scheduler Scheduled Job Complete: erpnext.healthcare.doctype.patient_appointment.patient_appointment.send_appointment_reminder for erpnext
2022-06-30 08:28:06,601 INFO scheduler Scheduled Job Start: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 08:28:06,931 INFO scheduler Scheduled Job Complete: erpnext.projects.doctype.project.project.project_status_update_reminder for erpnext
2022-06-30 08:28:07,007 INFO scheduler Scheduled Job Start: frappe.monitor.flush for erpnext
2022-06-30 08:28:07,161 INFO scheduler Scheduled Job Complete: frappe.monitor.flush for erpnext
2022-06-30 08:28:07,228 INFO scheduler Scheduled Job Start: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 08:28:07,387 INFO scheduler Scheduled Job Complete: frappe.utils.global_search.sync_global_search for erpnext
2022-06-30 08:28:07,465 INFO scheduler Scheduled Job Start: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 08:28:07,660 INFO scheduler Scheduled Job Complete: frappe.integrations.doctype.razorpay_settings.razorpay_settings.capture_payment for erpnext
2022-06-30 08:28:07,737 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 08:28:07,945 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.notify_unreplied for erpnext
2022-06-30 08:28:08,017 INFO scheduler Scheduled Job Start: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 08:28:08,218 INFO scheduler Scheduled Job Complete: frappe.email.doctype.email_account.email_account.pull for erpnext
2022-06-30 08:28:08,287 INFO scheduler Scheduled Job Start: frappe.email.queue.flush for erpnext
2022-06-30 08:28:08,455 INFO scheduler Scheduled Job Complete: frappe.email.queue.flush for erpnext
2022-06-30 08:29:05,974 DEBUG scheduler Access denied for site demo
2022-06-30 08:29:06,162 DEBUG scheduler Queued events for site erpnext
2022-06-30 08:29:06,164 DEBUG scheduler Access denied for site test
2022-06-30 08:29:06,166 DEBUG scheduler Access denied for site test.dev
2022-06-30 08:30:06,416 DEBUG scheduler Access denied for site demo
2022-06-30 08:30:06,607 INFO scheduler Scheduled Job Start: erpnext.utilities.doctype.video.video.update_youtube_data for erpnext
2022-06-30 08:30:06,697 DEBUG scheduler Queued events for site erpnext
